server/backend/main.py

```diff
@@ -0,0 +1,231 @@
+import asyncio
+import logging
+import os
+import re
+from itertools import islice
+
+import redis
+from fastapi import FastAPI, HTTPException
+from starlette.middleware.cors import CORSMiddleware
+from fastapi_redis_cache import FastApiRedisCache, cache
+from rq import Queue
+from rq.job import Job
+from rq.exceptions import NoSuchJobError
+
+from .config import get_config_values
+
+logger = logging.getLogger(__name__)
+
+app = FastAPI()
```

Comment:
Is this suppose to be twice?

Reply:
Oh, wow, definitely not! 🤦  Thanks for pointing that out.